files/KoreBuild/modules/vstest/module.targets

```diff
@@ -115,10 +115,10 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
       Condition="'@(BenchmarksToValidate)' != ''" />
 
     <MSBuild Projects="%(BenchmarksToValidate.Identity)" Targets="GetTargetPath" Properties="$(SolutionProperties)">
-      <Output TaskParameter="TargetOutputs" PropertyName="ExecPath" />
+      <Output TaskParameter="TargetOutputs" ItemName="BenchmarkExecPath" />
     </MSBuild>
 
-    <RunDotNet Command="$(ExecPath) * --validate-fast" />
+    <RunDotNet Command="&quot;%(BenchmarkExecPath.Identity)&quot; * --validate-fast" />
```

Comment:
Consider using the `Arguments` parameter instead, which takes `string[]`, so you don't have to manually do the escaping.
```xml
<RunDotNet Arguments="%(BenchmarkExecPath.Identity);*;--validate-fast" />
```